Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -660,6 +644,36 @@ namespace AzToolsFramework
                 }
                 const auto valueAtSubPath = GetDPE()->GetAdapter()->GetContents()[subPath];
 
+                if (!childWidget)
+                {
+                    // if there's a null entry in the current place for m_domOrderedChildren,
+                    // that's ok if this entry isn't expanded to that depth and need not follow the change any further
+                    // if we are expanded, then this patch references an unsupported handler, which might a problem
+                    if (IsExpanded())
+                    {
+                        // widget doesn't exist, but maybe we can make one now with the known contents
+                        auto handlerId =
+                            AZ::Interface<PropertyEditorToolsSystemInterface>::Get()->GetPropertyHandlerForNode(valueAtSubPath);
+
+                        if (handlerId)
+                        {
+                            // have a proper handlerID now, see if we can make a widget from this value now
+                            auto replacementWidget = CreateWidgetForHandler(handlerId, valueAtSubPath);
+                            if (replacementWidget)
+                            {
+                                AddColumnWidget(replacementWidget, childIndex, valueAtSubPath);
+                                AddDomChildWidget(childIndex, replacementWidget);
+                            }
+                        }
+                        else
+                        {
+                            AZ_Warning("Document Property Editor", false, "got patch for unimplemented PropertyHandler");
```

Comment:
Should we make this an `AZ_Error` instead to raise a bigger red flag? Once we have parity with all the currently supported handlers it seems like this would only happen if the user specified a new custom handler but didn't register it?

Reply:
I wish! I did that a long time ago, but it turns out that there are all sorts of unimplemented Handlers and we just need to let them go, right @NicholasVanSickle ?